verticapy/learn/model_selection.py

```diff
@@ -1773,36 +1776,43 @@ def grid_search_cv(
     if training_score:
         result = tablesample(
             {
-                "parameters": [elem[0] for elem in data],
-                "avg_score": [elem[1] for elem in data],
-                "avg_train_score": [elem[2] for elem in data],
-                "avg_time": [elem[3] for elem in data],
-                "score_std": [elem[4] for elem in data],
-                "score_train_std": [elem[5] for elem in data],
+                "parameters": [d[0] for d in data],
+                "avg_score": [d[1] for d in data],
+                "avg_train_score": [d[2] for d in data],
+                "avg_time": [d[3] for d in data],
+                "score_std": [d[4] for d in data],
+                "score_train_std": [d[5] for d in data],
             }
         )
         if print_info and (
             "final_print" not in kwargs or kwargs["final_print"] != "no_print"
         ):
             print("\033[1mGrid Search Selected Model\033[0m")
             print(
-                f"{str(estimator.__class__).split('.')[-1][:-2]}; Parameters: {result['parameters'][0]}; \033[91mTest_score: {result['avg_score'][0]}\033[0m; \033[92mTrain_score: {result['avg_train_score'][0]}\033[0m; \033[94mTime: {result['avg_time'][0]}\033[0m;"
+                f"{str(estimator.__class__).split('.')[-1][:-2]}; "
+                f"Parameters: {result['parameters'][0]}; \033"
+                "[91mTest_score: {result['avg_score'][0]}\033[0m;"
+                " \033[92mTrain_score: {result['avg_train_score']"
+                "[0]}\033[0m; \033[94mTime: {result['avg_time'][0]}\033[0m;"
```

Comment:
shouldn't there be f in the start of the strings?

Reply:
yes, that's a mistake.